src/components/FileSelector/styled/index.js

```diff
@@ -172,8 +180,20 @@ export const StyledIconContainer = attachThemeAttrs(styled(IconContainer))`
             width: 64px !important;
             height: 64px !important;
             font-size: 64px !important;
+
+            ${props.error &&
+                `
+                height: 40px !important;
+                width: 40px !important;
+                font-size: 40px !important;
+                color: ${props.palette.error.main};
+                
+                #OvalPartialShape {
```

Comment:
we should not overcomplicate this if we use these styles `fill="#000" fillOpacity=".102"` on the `path` element we don't need to use this.
using the above styles we guaranty these color always will be darker

Reply:
sure, no probs, keep it simple